packages/dev/core/src/Meshes/meshUVSpaceRenderer.ts

```diff
@@ -171,16 +221,47 @@ export class MeshUVSpaceRenderer {
             engine.clear(this.clearColor, true, true, true);
             engine.unBindFramebuffer(this.texture.renderTarget);
         }
+        if (this._finalPostProcess?.inputTexture) {
+            const engine = this._scene.getEngine();
+
+            engine.bindFramebuffer(this._finalPostProcess?.inputTexture);
+            engine.clear(this.clearColor, true, true, true);
+            engine.unBindFramebuffer(this._finalPostProcess?.inputTexture);
+        }
     }
 
     /**
-     * Disposes of the ressources
+     * Disposes of the resources
      */
     public dispose() {
         if (this._textureCreatedInternally) {
             this.texture.dispose();
             this._textureCreatedInternally = false;
         }
+        this._userCreatedTextureConfigured = false;
+        this._maskTexture?.dispose();
+        this._maskTexture = null;
+        this._finalPostProcess?.dispose();
+        this._finalPostProcess = null;
+    }
+
+    private _configureUserCreatedRTT(): void {
+        this._userCreatedTextureConfigured = true;
+        if (MeshUVSpaceRenderer._IsRenderTargetTexture(this.texture)) {
+            this.texture.setMaterialForRendering(this._mesh, MeshUVSpaceRenderer._GetShader(this._scene));
+            this.texture.onClearObservable.addOnce(() => {
+                this._scene.getEngine().clear(this.clearColor, true, true, true);
+                if (MeshUVSpaceRenderer._IsRenderTargetTexture(this.texture)) {
+                    this.texture.onClearObservable.add(() => {});
+                }
+            });

```

Comment:
As far as I'm concerned, it's up to the user whether or not to clear the RTT (after all, he may want to start with an uncleared texture). You can therefore replace this code with :
```typescript
this.texture.onClearObservable.add(() => {});
```